L1Trigger/L1TMuonOverlap/src/XMLConfigWriter.cc

```diff
@@ -36,6 +38,25 @@ XERCES_CPP_NAMESPACE_USE
 #endif
 //
 
+namespace {
+ unsigned int eta2Bits(unsigned int eta) {
```

Comment:
I would suggest to use `0b100000000` rather than `bitset<9>("100000000").to_ulong()` , etc.